.gitignore

```diff
@@ -67,3 +67,7 @@ ssl/
 out/
 .idea_modules/
 
+# Monitoring-Setup
+prometheus/prometheus_db/*
+!prometheus/prometheus_db/.gitkeep
```

Comment:
@burnerlee the `.gitkeep` files are not being ignored. Please fix that too.

Reply:
@Ram81 I committed these files to push the grafana/grafana_db, prometheus/prometheus_db folders, as these directories are currently empty, i need to keep a gitkeep file in them and commit that file

Reply:
@burnerlee we don't need to check in the folders if there is nothing in that folder. And you can remove `.gitkeep` from `promethues` folder as we already have the config file in that folder.